README.md

```diff
@@ -42,6 +42,28 @@ For example:
 When filing bug reports with unit test output, always use Maven
 option `-B` to avoid control characters in output.
 
+## Docker image
+
+At Docker Hub:
+
+* https://hub.docker.com/u/folioorg/okapi released versions
```

Comment:
Looks like the correct link should be `https://hub.docker.com/r/folioorg/okapi`. @dcrossleyau, can you confirm? If yes, we need to update https://dev.folio.org/download/artifacts/ as well.

Reply:
Yes, good catch. I should have copied the complete URL, not appending `/okapi` to the `https://hub.docker.com/u/folioorg` URL.
Fixed.

Reply:
I fixed the namespace URLs at dev.f.o